### DISCUSSION TITLE
3.23.61 impossible to open side bar

### DISCUSSION THREAD

**Discussion by andy-goryachev** [standardnotes/forum#2851] [General]
Once collapsed, it is impossible to open the side bar:

![2022-08-02 22_29_17-Standard Notes](https://user-images.githubusercontent.com/5430849/182531093-970fc351-1118-44f6-a680-6fdac5df3860.png)

Perhaps it should not collapse completely, but collapse to a set of icons, as the case is in many apps (outlook).

**Comment by myreli** [comment]
Double-clicking the sidebar margin would open in that case, but I believe your suggestion is now the behaviour:

<img width="425" alt="image" src="https://user-images.githubusercontent.com/17554234/221364602-004e9409-b01e-4a84-b41b-2bdd2ea9030b.png">

**Comment by andy-goryachev** [comment]
it is still impossible to open side bar once collapsed on Windows desktop app.  This is still a bug.  Tested with 3.150.0 version.

**Comment by andy-goryachev** [comment]
Please convert this to a bug.

**Comment by myreli** [comment]
Hi, @andy-goryachev thank you for testing it again. I just labelled it as a `bug`. We are converting all issues into Discussions to better manage the forum, and this is a community effort in triaging older issues. 


Could you try these toggles to see if you can show and hide the sidebar?
<img width="320" alt="image" src="https://user-images.githubusercontent.com/17554234/221373647-8e24b098-a538-4224-965f-ca773c36df3d.png">

**Comment by andy-goryachev** [comment]
These toggles do not help.  Perhaps an item or a toggle should be added to "Display options menu", though I would prefer to be able to achieve this with a single click.

Also, it would be nice if the support requests from paying members would be labeled as such and prioritized.

Thank you.

**Comment by andy-goryachev** [comment]
created a bug https://github.com/standardnotes/forum/issues/2871
